src/server/scripts/Kalimdor/RuinsOfAhnQiraj/boss_ayamiss.cpp

```diff
@@ -253,29 +234,6 @@ struct npc_hive_zara_larva : public ScriptedAI
         }
     }
 
-    void MoveInLineOfSight(Unit* who) override
-    {
-        if (_instance->GetBossState(DATA_AYAMISS) == IN_PROGRESS)
-            return;
-
-        ScriptedAI::MoveInLineOfSight(who);
-    }
-
-    void AttackStart(Unit* victim) override
-    {
-        if (_instance->GetBossState(DATA_AYAMISS) == IN_PROGRESS)
-            return;
-
-        ScriptedAI::AttackStart(victim);
-    }
-
-    void UpdateAI(uint32 diff) override
-    {
-        if (_instance->GetBossState(DATA_AYAMISS) == IN_PROGRESS)
-            return;
-
-        ScriptedAI::UpdateAI(diff);
-    }
```

Comment:
What's the reason for dropping all of these?
Or is this replaced by setting it to "Passive" initially

Reply:
Yes.